package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "tg-wakatime-bot",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/mohammadizanloo55/tg-wakatime-bot.git",
+  "author": "mohammadizanloo55 <mohammadizanloo55@gmail.com>",
+  "license": "MIT",
+  "devDependencies": {
+    "@fullstacksjs/eslint-config": "^6.5.2",
```

Comment:
Please configure the package manager not to use the `^` save prefix.
Also, we use npm in our community. So it's better to keep consistency.